### PR TITLE
Moves SwiftLint to Danger plugin.

### PR DESCRIPTION
**Pull request by ashfurrow**
https://github.com/ashfurrow/danger-swiftlint

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,11 +3,9 @@ osx_image: xcode8.3
 
 before_script:
   - bundle install
-  - cd Bootstrap ; bundle exec pod install --repo-update ; cd ..
+  - cd Bootstrap ; bundle exec pod install ; cd ..
```
Comment (marcelofabri) [inline .travis.yml]: this change can introduce failures, can't it?

Comment (orta) [inline .travis.yml]: Yeah, there _could_ be mismatches during version updates

Comment (ashfurrow) [inline .travis.yml]: It's possible, but we can add it back in later if we need to. I took it out to save time on CI, now I'm not sure it's worth it. Any thoughts?

Comment (orta) [inline .travis.yml]: I'd live with it till something breaks, it would possibly happen when you update your RxSwift deps for example

**Pull request merged**